lib/mongo_client.js

```diff
@@ -411,8 +439,18 @@ MongoClient.connect = function(url, options, callback) {
  * @param {SessionOptions} [options] optional settings for a driver session
  * @return {ClientSession} the newly established session
  */
+const startSessionSchema = {
+  // TODO
+  explicit: { type: 'boolean', default: true }
```

Comment:
I don't think `explicit` is something we actually want to expose. We just use it to mark internally whether it is an explicitly created or implicitly  created session.